### PR TITLE
Added initial code for insert  micro benchmarks

### PR DESCRIPTION
**Pull request by pluraliseseverythings**

### CODE REVIEW

**Comment by pluraliseseverythings** [comment]
!rtg

File: grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java [hunk 1]
```diff
@@ -141,6 +139,9 @@ public static void main(String[] args) {
     }
 
     public void start() {
+        redisWrapper.testConnection();
+        LOG.info("Starting task manager {}", taskManager.getClass().getCanonicalName());
+        taskManager.start();
```
Comment (pluraliseseverythings) [inline grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java]: Moved start of task manager to start from constructor

**Comment by pluraliseseverythings** [comment]
!rtg

File: Jenkinsfile [hunk 2]
```diff
@@ -2,97 +2,100 @@
...
+                            sh '../grakn-test/test-snb/src/generate-SNB/load-SNB.sh arch validate'
+                        }
+                    }
+                    stage('Measure Size') {
+                        sh 'nodetool flush'
+                        sh 'du -hd 0 grakn-package/db/cassandra/data'
+                    }
+                }
+                timeout(360) {
+                    stage('Run the benchmarks') {
```
Comment (pluraliseseverythings) [inline Jenkinsfile]: This is the only bit I added, the rest is formatting the indentation

File: grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java [hunk 3]
```diff
@@ -263,11 +256,12 @@ void sendQueriesToLoader(Collection<Query> queries){
         Retryer<TaskResult> sendQueryRetry = RetryerBuilder.<TaskResult>newBuilder()
                 .retryIfExceptionOfType(IOException.class)
                 .retryIfRuntimeException()
+                .retryIfResult(r -> r != null && r.getCode().startsWith("5"))
```
Comment (pluraliseseverythings) [inline grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java]: I'm fixing this awfulness next. I'll try to put a benchmark on REST and change the http client to something that doesn't require us to do the retry explicitly

File: grakn-test/test-integration/src/test/java/ai/grakn/test/engine/postprocessing/PostProcessingTaskTest.java [hunk 4]
```diff
@@ -52,7 +52,7 @@
```
Comment (pluraliseseverythings) [inline grakn-test/test-integration/src/test/java/ai/grakn/test/engine/postprocessing/PostProcessingTaskTest.java]: From here it's just renaming (that method doesn't actually start anything)

File: grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java (hunk 3 above)
Comment (aelred) [reply grakn-client/src/main/java/ai/grakn/client/BatchMutatorClient.java]: thanks for apologising

**Pull request merged**